src/lib.rs

```diff
@@ -161,8 +161,6 @@ pub use bytes;
 pub use futures;
 #[doc(inline)]
 pub use multiaddr;
-#[doc(inline)]
-pub use multihash;
```

Comment:
What is the motivation for removing this re-export in favor of adding a re-export from `libp2p-core`? It would be good to avoid unnecessary API changes, unless you think there is a good reason to do so.

Reply:
The main motivation was, that rust-multihash makes heavy use of features and I thought having to get those right in a single `Cargo.toml` file is easier. That's the reason I re-exported in `libp2p-core`, so that other libp2p crates can use it easily (as the ones using Multihash all depend on `libp2p-core` anyway). I overlooked that I break the public API of `libp2p`.

I see several options, please let me know which one to pick:

 1. Keeping this change
 2. re-exporting `multihash` with its default features
 3. re-exporting `multihash` with the same features defined as in `libp2p-core`
 4. re-exporting `multihash` from `libp2p-core`.

Reply:
How about `3 + 4`, i.e. just doing
```rust
pub use core::multihash;
```
here?

Reply:
Yes that is what I meant with `4.`. I'll do that.